hswidget/fonttablewidget/state.go

```diff
@@ -0,0 +1,80 @@
+package fonttablewidget
+
+import (
+	"fmt"
+
+	"github.com/ianling/giu"
+)
+
+type fontTableWidgetMode int
+
+const (
+	// viewer state
+	fontTableWidgetViewer fontTableWidgetMode = iota
+	// edit rune
+	fontTableWidgetEditRune
+	// opens a dialog and adds a new item on first free index
+	fontTableWidgetAddItem
```

Comment:
since this is in its own package, you can reduce the noise
```golang
const (
	modeViewer fontTableWidgetMode = iota
	modeEditRune
	modeAddItem
)
```

Reply:
solved in 18e5eef